main.nf

```diff
@@ -14,7 +14,6 @@ def helpMessage() {
     log.info"""
     Usage:
     The typical command for running the pipeline is as follows:
-
```

Comment:
All of these line keep on being deleted for some reason 😅 Can we add them back in please.

Reply:
Hi @drpatelh, we have added those empty lines back in.